shared/javascripts/options.js

```diff
@@ -63,6 +63,16 @@ if (Privly === undefined) {
   }
   Privly.options = {};
 
+  // CommonJS Module
+  if (typeof module !== "undefined" && module.exports) {
+    module.exports.options = Privly.options;
+    // load dependencies
+    ls = require("./local_storage.js").ls;
+    Privly.storage = require("./storage.js").storage;
```

Comment:
This script is loaded [here](https://github.com/hitesh96db/privly-jetpack/blob/options_interface/lib/privly_ns.js#L8) as a CommonJS module.
Every script needs to load it's own dependencies. Won't there be a cross-dependency(as in circular reference) issue ?